packages/chart/src/FigureChartModel.ts

```diff
@@ -157,6 +157,9 @@ class FigureChartModel extends ChartModel {
     const { charts } = this.figure;
     const axisTypeMap = ChartUtils.getAxisTypeMap(this.figure);
     const activeSeriesNames: string[] = [];
```

Comment:
Side note - this could be optimized by using a `Set<string>` instead of a `string[]`.